elyra/pipeline/processor_airflow.py

```diff
@@ -223,6 +223,7 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                              'cpu_request': operation.cpu,
                              'mem_request': operation.memory,
                              'gpu_request': operation.gpu,
+                             'source': operation.component_params['filename'],
```

Comment:
A rename may be in order for `source` here. Should be more specific (I don't wan't another `type` issue 😄 )